<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBPic</class>
 <widget class="QWidget" name="DBPic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DBPic</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnImport">
       <property name="text">
        <string>导入图片(&amp;I)</string>
       </property>
       <property name="icon">
        <iconset resource="skdraw.qrc">
         <normaloff>:/images/import</normaloff>:/images/import</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExport">
       <property name="text">
        <string>导出图片(&amp;E)</string>
       </property>
       <property name="icon">
        <iconset resource="skdraw.qrc">
         <normaloff>:/images/export</normaloff>:/images/export</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDel">
       <property name="text">
        <string>删除图片(&amp;D)</string>
       </property>
       <property name="icon">
        <iconset resource="skdraw.qrc">
         <normaloff>:/images/close</normaloff>:/images/close</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>选中图片(&amp;O)</string>
       </property>
       <property name="icon">
        <iconset resource="skdraw.qrc">
         <normaloff>:/images/ok</normaloff>:/images/ok</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>取消(&amp;C)</string>
       </property>
       <property name="icon">
        <iconset resource="skdraw.qrc">
         <normaloff>:/images/delete</normaloff>:/images/delete</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="skdraw.qrc"/>
 </resources>
 <connections/>
</ui>
